content/actions/reference/events-that-trigger-workflows.md

```diff
@@ -655,7 +655,7 @@ on:
 
 {% data reusables.webhooks.workflow_run_desc %}
```

Comment:
It looks like this section is missing the table that the other event sections have. We should add that rather than including the information in the description:

```suggestion
{% data reusables.webhooks.workflow_run_desc %}

| Webhook event payload | Activity types | `GITHUB_SHA` | `GITHUB_REF` |
| --------------------- | -------------- | ------------ | -------------|	
| [`workflow_run`](/webhooks/event-payloads/#workflow_run) | - n/a | Last commit on default branch | Default branch |	
```